test/e2e/specs/change-detection.test.js

```diff
@@ -94,7 +94,7 @@ describe( 'Change detection', () => {
 		// Toggle post as sticky (not persisted for autosave).
 		await ensureSidebarOpened();
 
-		const postStickyToggleButton = ( await page.$x( "//label[contains(text(), 'Stick to the Front Page')]" ) )[ 0 ];
+		const postStickyToggleButton = ( await page.$x( "//label[contains(text(), 'Pending Review')]" ) )[ 0 ];
```

Comment:
Authors can mark things needing review, they can't stick things to the front page. The effect in the test is the same.

Reply:
If I'm reading correctly, the E2E tests can still be run by a non-author, in which case the label would not have the changed effect here. Should this be made more generic? Lookup element by its class? A helper utility which returns the button accounting for role?

Reply:
As far as I can see, no matter what your user role, clicking "Pending Review" marks the post as dirty, and that's what's getting tested here.

Reply:
Oh, I think I misunderstood the reason for the change. It’s just changing it to an option which is visible for both admins and authors.